mgrpxy/cmd/squid/squid.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2024 SUSE LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package squid
+
+import (
+	"github.com/spf13/cobra"
+	. "github.com/uyuni-project/uyuni-tools/shared/l10n"
+	"github.com/uyuni-project/uyuni-tools/shared/types"
+)
+
+// NewCommand entry command for managing squid cache.
+// Setup for subcommand to clear (the cache).
+func NewCommand(globalFlags *types.GlobalFlags) *cobra.Command {
+	var squidCmd = &cobra.Command{
+		Use:   "squid",
+		Short: L("Manage Squid cache"),
+		Long:  L("Manage Squid cache"),
+		Run: func(cmd *cobra.Command, args []string) {
+			_ = cmd.Help()
+		},
```

Comment:
I would not have `squid` in the command name, but something more generic like:

```suggestion
		Use:   "cache",
		Short: L("Manage proxy cache"),
		Long:  L("Manage proxy cache"),
		Run: func(cmd *cobra.Command, args []string) {
			_ = cmd.Help()
		},
```

Maybe later we'll add a `--squid` parameter to `mgrpxy cache clear` to target which cache to clear.  There are different caches in a proxy. Even though the squid one is usually to one to clear, we shouldn't close the door to the others.

Reply:
I see your point. Merged.